### PR TITLE
Security fixes-Babel Security vulnerability

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
Changes 
`babel-minify-webpack-plugin` has been archived ( https://github.com/webpack-contrib/babel-minify-webpack-plugin?tab=readme-ov-file ) so its been updated it to the new package `terser-webpack-plugin` (https://github.com/webpack-contrib/terser-webpack-plugin) to solve the issue of `babel-traverse.` As on running `npm i `webpack was having a conflict with `eslint`, the version of` eslint` was downgraded from `eslint: ^9.3.0` to `eslint: ^7.32.0`

### CODE REVIEW

**Comment by vishal-deriv** [comment]
Looks good from security side.
It fixed both dependabot issues on JSON and babel subdependency.

File: src/index.js [hunk 1]
```diff
@@ -1,83 +1,90 @@
-import 'core-js/stable';
```
Comment (amina-deriv) [inline src/index.js:1]: can u check if these  formating changes are required?

File: src/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default class DocumentUploader {
         this.clients = {};
         this.wrapConnection();
     }
+
```
Comment (amina-deriv) [inline src/index.js:13]: lets remove these prettier changes

File: package.json [hunk 3]
```diff
@@ -1,77 +1,79 @@
...
+	  "base64-js": "^1.2.1",
+	  "eslint": "^7.32.0",
+	  "eslint-config-airbnb-base": "^11.3.2",
+	  "eslint-config-binary": "^1.0.2",
+	  "eslint-config-prettier": "^2.3.0",
+	  "eslint-plugin-import": "^2.7.0",
+	  "jest": "^29.7.0",
+	  "lint-staged": "^4.0.3",
+	  "prettier-eslint-cli": "^5.0.0",
+	  "terser-webpack-plugin": "^5.3.10",
```
Comment (Dhruv-deriv) [inline package.json:65]: added line here ^

File: package.json [hunk 4]
```diff
@@ -1,77 +1,79 @@
...
+	  "webpack": "^5.91.0",
+	  "webpack-cli": "^5.1.4",
```
Comment (Dhruv-deriv) [inline package.json:67]: "webpack-cli": "^5.1.4", here as well

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**